runtime/vm/createramclass.cpp

```diff
@@ -2800,25 +2811,19 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 					|| ((LOAD_LOCATION_PATCH_PATH == locationType)
 					|| (LOAD_LOCATION_MODULE == locationType))
 				) {
-					U_8* packageUTF = NULL;
-					J9ROMClass *packageRomClass = romClass;
-					U_32 pkgNameLength = 0;
-
-					if (J9ROMCLASS_IS_ARRAY(romClass)) {
-						packageRomClass = elementClass->romClass;
-					}
-
-					packageUTF = J9UTF8_DATA(J9ROMCLASS_CLASSNAME(packageRomClass));
-					pkgNameLength = (U_32) packageNameLength(packageRomClass);
+					U_32 pkgNameLength = (U_32) packageNameLength(romClass);
```

Comment:
I forgot  to remove some of the code that as used in the array case. I have fixed it to make it more clear.